app/assets/stylesheets/pages/stories.scss

```diff
@@ -59,6 +59,24 @@
       font-weight: 400;
     }
 
+    .question input[type="radio"] {
+      display:none;
+    }
+
+    .question label {
+        display:inline-block;
+        background-color:#ddd;
+        padding: 10px 20px;
+        font-size:16px;
+        border: 2px solid #444;
+        border-radius: 4px;
+    }
+
+    .question input[type="radio"]:checked + label {
+        background-color:#bfb;
+        border-color: #4c4;
+    }
+
```

Comment:
These labels aren't compatible with the design guidelines (rounded edges, color choice, thick border).

Reply:
These three styles could be refactored into a nest inside `.question {}`

Reply:
You should add `cursor: pointer` to show clickability.

Reply:
Where are the design guidelines (as in where do they live)? I have never seen them before. Can you provide more information on what changes would allow this pull request to meet the guidelines?

Reply:
rounded edges -> don't use rounded edges, 
color choice -> use standardized colors from the palette
thick border -> remove the border

These guidelines were distributed multiple times at the beginning of the project and we had many contentious conversations among the team regarding their implementation. They are located on the K drive in one of the project folders at this point.